api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -467,7 +468,6 @@ private void verifyInvitationKey(String invitationKey){
     if (updatedProfile.getContactEmail() != null) {
       if (!updatedProfile.getContactEmail().equals(user.getContactEmail())) {
         mailChimpService.addUserContactEmail(updatedProfile.getContactEmail());
-        user.setEmailVerificationStatus(EmailVerificationStatus.PENDING);
```

Comment:
Shouldn't changing your email "unverify" your email? Possibly just a TODO for now, but I think it still needs to be handled somehow.

Reply:
I believe this requires more refinement. We have a story that addresses changing contact email and I propose we define requirements for that story.

Reply:
Sounds good.

Reply:
For reference, the contact email change story is https://precisionmedicineinitiative.atlassian.net/browse/RW-719